src/Uno.UI/Microsoft/UI/Xaml/Controls/AppBar/AppBar.Partial.cs

```diff
@@ -0,0 +1,1701 @@
+﻿#nullable enable
```

Comment:
Add the copyright comment and source file here and in other ported files (see for example `\src\Uno.UI\UI\Xaml\Controls\Primitives\ButtonBase\ButtonBase.mux.cs`)